include/circt/Dialect/Arc/ArcOps.td

```diff
@@ -659,11 +680,12 @@ def ModelOp : ArcOp<"model", [RegionKindInterface, IsolatedFromAbove,
     specifies the I/O of the module associated to this model.
   }];
   let arguments = (ins SymbolNameAttr:$sym_name,
-                       TypeAttrOf<ModuleType>:$io);
+                       TypeAttrOf<ModuleType>:$io,
+                       OptionalAttr<FlatSymbolRefAttr>:$initialFn);
```

Comment:
SymbolUserOpInterface is necessary if we want to add initializer as a function.

Reply:
Thanks! Missed that one.